File: app/src/main/java/com/game/kotvitz/war/GameMedia.java
package com.game.kotvitz.war;

import android.content.Context;
import android.media.MediaPlayer;

public class GameMedia {

    public void playClickSound(Context context) {
        final MediaPlayer mp = MediaPlayer.create(context, R.raw.click);
        mp.start();
    }

    public void playSwitchSound(Context context) {
        final MediaPlayer mp = MediaPlayer.create(context, R.raw.switchie);
        mp.start();
    }
}

File: app/src/main/java/com/game/kotvitz/war/GameMedia.kt
package com.game.kotvitz.war

import android.content.Context
import android.media.MediaPlayer

class GameMedia(private val context: Context) {

    fun playClickSound() {
        val mp = MediaPlayer.create(context, R.raw.click)
        mp.start()
    }

    fun playSwitchSound() {
        val mp = MediaPlayer.create(context, R.raw.switchie)
        mp.start()
    }

    fun playCardSlideSound() {
        val mp = MediaPlayer.create(context, R.raw.card_slide)
        mp.start()
    }

    fun playCardPlaceSound() {
        val mp = MediaPlayer.create(context, R.raw.card_place)
        mp.start()
    }
}
